datasets/za/mm_executive_mayors/za_mm_executive_mayors.yaml

```diff
@@ -0,0 +1,51 @@
+title: "South Africa Municipal Leadership"
+url: https://municipalmoney.gov.za/profiles/municipality-NC076-thembelihle/
+entry_point: crawler.py
+prefix: za-treasury
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+coverage:
+  frequency: daily
+  start: 2024-08-20
+summary: >
+  Profiles of municipal heads in South Africa, focusing on key political figures 
+  such as the Mayor and Municipal Manager.
+description: |
+  The dataset provides information on Politically Exposed Persons (PEPs) across all 
+  municipalities in South Africa. It focuses on individuals holding key positions 
+  of influence, including Mayor/Executive Mayor and Municipal Manager, within each 
+  municipality.
+
+publisher:
+  name: National Treasury of South Africa
+  acronym: NT
+  official: true
+  description: |
+    The National Treasury of South Africa manages the national government's finances, 
+    ensures transparency and accountability, coordinates macroeconomic policy, and supports 
+    resource allocation to reduce poverty and vulnerability among marginalized populations.
+    [Source: Official Website](https://nationalgovernment.co.za/units/view/27/national-treasury)
```

Comment:
Best to use an official (.gov.x) source for official source descriptions, rather than a random .co.za

```suggestion
     The Constitution of the Republic (Chapter 13) mandates the National Treasury to ensure
     transparency, accountability and sound financial controls in the management of public finances.
```